h2o-r/h2o-package/R/frame.R

```diff
@@ -3782,6 +3782,149 @@ h2o.arrange <- function(x, ...) {
   .newExpr("sort", x, by-1L, as.numeric(ascend))
 }
 
+generateColInd <-function(data, by) {
```

Comment:
is this an internal function?  It's not documented and exported (so it won't show up in R documentation).  Either way, probably change the name to `generate_col_index`